instrumentation/gwt-2.0/javaagent/src/testapp/java/test/gwt/Greeting.gwt.xml

```diff
@@ -3,6 +3,9 @@
 
   <entry-point class="test.gwt.client.GreetingEntryPoint" />
 
+  <!-- only generate one permutation to make the build faster -->
+  <set-property name="user.agent" value="safari" />
```

Comment:
Why safari if test uses chrome?

Reply:
GWT is from way before Chrome :) Back then it was all Webkit and I suspect it serves safari pages to Chrome.

Reply:
Possible values are here https://gwt.googlesource.com/gwt/+/master/user/src/com/google/gwt/useragent/UserAgent.gwt.xml#21